### PR TITLE
SEO-181942-UG-Duplicate-title

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Kindly review the duplicate title changes.

Regards,
Edith.

### CODE REVIEW

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/annotation/text-markup-annotation.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Text Markup Annotation in ##Platform_Name## Pdfviewer Component
+title: Text Markup Annotation Guide in ##Platform_Name## Pdfviewer Component | Syncfusion
```
Comment (MallikaRK) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/annotation/text-markup-annotation.md:3]: is this meaningful? remove guide here.
Check the control name in FT page and update.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/annotation/text-markup-annotation.md (hunk 1 above)
Comment (Edithabich) [reply ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/annotation/text-markup-annotation.md:3]: resolved

Comment (MallikaRK) [reply ej2-asp-core-mvc/pdfviewer/EJ2_ASP.MVC/annotation/text-markup-annotation.md:3]: Hi @Edithabich , still my comment is not addressed. Can you remove guide word in Title? also you can use "Support" word here.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
commented

**Review by MallikaRK — APPROVED** [review]
approved

**Pull request merged**